docs/source/manual/configuration.rst

```diff
@@ -105,13 +105,15 @@ Request Log
 .. code-block:: yaml
 
     server:
-      requestLog: 
-        timeZone: UTC
+      requestLog:
+        - type: SLF4J
```

Comment:
Agree. We should mark the `logback-access` implementation as default, so users should not set the type explicetly.
But this should be not relevant after removing of the `slf4j` request log.